app/input_objects/account/organisation_input.rb

```diff
@@ -22,9 +22,7 @@ def assign_form_values
 private
 
   def log_organisation_chosen_event
-    EventLogger.log({
-      event: "organisation_chosen",
-      user_id: user.id,
+    Rails.logger.info("User chose their organisation", {
```

Comment:
Please check with Anne whether is affects her reporting if this message content and key changes; I ask because if it's currently in the `event` field then it might be being used.

Reply:
On that note, can we mark whatever fields Anne is relying on with some sort of comment pls?

Reply:
I've checked with Anne and she doesn't use this log lines for any reporting.

What Me and Dan were thinking is that in forms-runner we could potentially move towards treating anything logged using the `EventLogger` as something we shouldn't be changed as it will be used for analytics purposes.
If we do this, we could then use `Rails.logger` for logging anything adhoc that isn't used for analytics and have a distinction there.

What do you reckon? Something we can maybe think of next time we're wanting to add more logging to runner.